# Patch created by Alexander Kriegisch (kriegaex, ip-phone-forum.de):
#  - Switch off syntax highlighting, saving 70 KB in file size
#  - If switched off, mc assumes we want b/w colour scheme by default. I do
#    not know why, but instead of forcing the user to call 'mc -c', I just
#    changed the default. B/w scheme can still be used via 'mc -b'.

--- edit/edit.h	2007-07-07 18:27:57.000000000 +0200
+++ edit/edit.h	2007-07-07 18:48:57.000000000 +0200
@@ -4,7 +4,8 @@
 #ifdef MIDNIGHT
 
 #ifdef HAVE_SLANG
-#define HAVE_SYNTAXH 1
+// Remove syntax highlighting to save some space (approx. 70 KB)
+// #define HAVE_SYNTAXH 1
 #endif
 
 #    include <stdio.h>
--- src/main.c	2007-07-07 20:58:02.000000000 +0200
+++ src/main.c	2007-07-07 20:58:06.000000000 +0200
@@ -304,7 +304,7 @@
 int midnight_colors [4];
 
 /* Force colors, only used by Slang */
-int force_colors = 0;
+int force_colors = 1;
 
 /* colors specified on the command line: they override any other setting */
 char *command_line_colors;
